data/forms/create-new-person.jsx

```diff
@@ -66,6 +69,20 @@ export default {
           label: 'Nationality',
           rules: { required: true },
         },
+        {
+          component: 'Select',
+          name: 'ethnicity',
+          label: 'Ethnicity',
+          options: Object.keys(ETHNICITIES),
+        },
+        {
+          component: 'Select',
+          name: 'macro_ethnicity',
```

Comment:
They should be the opposite, this one is going to indicate the final ethnicity id.

Reply:
🤦  understood it after I pushed up the changes, committed new changes now

Reply:
It's not really intuitive, it's actually a trick in order to not to have to change the data once submitted :)